ios/core/Sources/Types/Core/ConstantsController.swift

```diff
@@ -0,0 +1,31 @@
+import JavaScriptCore
+
+public class ConstantsController {
+    var constantsController: JSValue?
+
+    public func getConstants<T>(key: Any, namespace: String, fallback: Any? = nil) -> T? {
+        if let fallbackValue = fallback {
+            let value = self.constantsController?.invokeMethod("getConstants", withArguments: [key, namespace, fallbackValue])
+            return value?.toString() as? T
```

Comment:
actually i think we need `value?.toObject()` here because `T` can by `any` maybe add a test where getConstants returns an Int to verify it works?